src/main/java/seedu/parser/MainParser.java

```diff
@@ -29,6 +29,8 @@
 import seedu.exception.MissingDetailException;
 import seedu.exception.MissingIndexException;
 
+import java.lang.reflect.Array;
+
```

Comment:
Probably an unused import that was added when using print statements for testing the array, can consider removing if not required.